db/migrate/20160331232622_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
-  def change
-    create_table(:users) do |t|
```

Comment:
The easiest way would, which bypasses git craziness, is to just return to the master branch, make a new branch, and reimplement the same changes.

Alternatively you could make a new commit where you just explicitly add back in the file.  Either way works.